ourjapanlife/findadoc-frontend
# I18n package

## Resolves #195 

Uses the [package](https://github.com/ourjapanlife/findadoc-localization/packages/873560) instead of a submodule to incorporate latest i18n changes

### How to test

- Ensure the netlify build functions
- Pull the branch, run `yarn install`, then run the code
- You may need to `rm -rf i18n` from the root of your clone

### Screenshots
